Examples/ScaleMicrogrid/SolutionKeys.hpp

```diff
@@ -29,10 +29,11 @@
  * 
  * ODE Keys gives similar results.
  */
-constexpr real_type SCALE_MICROGRID_ERROR_TOL = 8.0e-5;
-constexpr real_type SCALE_MICROGRID_REL_TOL = 1.0e-7;
```

Comment:
The preceding comments are outdated and need to be updated. Tightening the tolerance does improve accuracy of the results.